### ISSUE TITLE
Steps for Installing this tool in Salesforce.

### ISSUE THREAD

**Issue by RahulP248** [Konstryktor/ApexUnusedMethodsFinder#2]
Hi there,

I have a salesforce org with huge number of classes and unused methods. I stumbled upon your post and was trying to use the code in my Salesforce org. I guess I'm missing some dependent components. Could you please share us the steps to use this tool in my salesforce org? It would be great help.

**Comment by eyewell** [comment]
yes. There is an object definition that is missing: ClassMethod__c. Fields include:

Apex_Classes__c,
Apex_Triggers__c;
Class_Name__c ;
Interfaces__c;
InUnusedClass__c
Method_Name__c ;
Modifiers__c ;
Number_of_Usage__c
Number_of_Usage__c ;
Parameters__c ;
Potential_Unused__c
Triggers_count__c
Type__c;
Unique_Field__c ;
Used__c,
Visualforce__c
Visualforce_Components__c
Visualforce_Pages__c
